### PR TITLE
refactor: Remove edgex-prefix from all service keys

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/.github/Contributing.md.

## What is the current behavior?
Service key constants have the `edgex-` prefix

## What is the new behavior?
Service key constants no longer have the `edgex-` prefix

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

BREAKING CHANGE: Service key names have changed.

## New Imports

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

**Comment by lenny-intel** [comment]
In draft until reading to push change through all services

File: clients/constants.go [hunk 1]
```diff
@@ -56,20 +56,19 @@ const (
...
-	SecurityBootstrapRedisKey           = "edgex-security-bootstrap-redis"
+	CoreCommandServiceKey               = "core-command"
+	CoreDataServiceKey                  = "core-data"
+	CoreMetaDataServiceKey              = "core-metadata"
+	SupportLoggingServiceKey            = "support-logging"
+	SupportNotificationsServiceKey      = "support-notifications"
+	SystemManagementAgentServiceKey     = "sys-mgmt-agent"
+	SupportSchedulerServiceKey          = "support-scheduler"
+	SecuritySecretStoreSetupServiceKey  = "security-secretstore-setup"
+	SecuritySecretsSetupServiceKey      = "security-secrets-setup"
```
Comment (jim-wang-intel) [inline clients/constants.go:67]: this one is already gone. should remove it.

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
This will impact the vault's service key and consul's service key based mechanism. so prepare to change that part too.

**Comment by jim-wang-intel** [comment]
Side note: Although this PR is not related to the service key change of consul, the current Consul's ACL policy rules definition is dependent on the node-name for the instance of `edgex-core-consul`  we are running: for example, the edgexPolicyRules is relying on node_prefix to be "edgex" so that it can confine the access of Consul from the outside world.  For that part, my suggestion is still to retain the "edgex-" prefix for Consul so that only Consul agent node edgex-xxx has written permission.  See details here: https://github.com/edgexfoundry/edgex-go/blob/master/internal/security/bootstrapper/command/setupacl/aclpolicies.go#L45-L47

File: clients/constants.go (hunk 1 above)
Comment (lenny-intel) [reply clients/constants.go]: Done

**Review by jim-wang-intel — APPROVED** [review]

**Review by iain-anderson — APPROVED** [review]

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**